kiali/ossm-kiali-assembly.adoc

```diff
@@ -14,5 +14,11 @@ include::modules/ossm-install-kiali-operator.adoc[leveloffset=1]
 
 include::modules/ossm-config-openshift-monitoring-kiali.adoc[leveloffset=1]
 
+[role="_additional-resources"]
+[id="additional-resources_{context}"]
+== Additional resources
```

Comment:
These workaround "Additional resources" are typically done with the "trailing include" version of the [Additional resources](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections) header:
```suggestion
[role="_additional-resources"]
.Additional resources
```